### DISCUSSION TITLE
Update count shows 1 in the badge with no updates to apply ?

### DISCUSSION THREAD

**Discussion by tonybyng** [statamic/cms#4632] [Q&A] [answered]
Hi

Ive got statamic installed with 2 addons - a listed and an unlisted (eloquenty). The "Updates" menu item on the left shows a [ 1 ] in red but when I click it says statamic and the one listed plugin is up to date and the other plugin is a composer based one and Ive updated that too. What could be causing it? Not sure if its related but there is a "Learn more about updates" link at the bottom that goes to a 404 so not sure if something is missing ?

Thanks.
Tony

**Comment by edalzell** [comment]
Have you tried a hard refresh in the browser? Perhaps something is cached.

**Reply by tonybyng** [reply]
@edalzell Thanks for the reply.  I have tried on a different browser with the same effect. Now Im getting a 2 rather than a 1 because of the recent Statamic update but when I select the Updates it only shows Statamic as needing an update, nothing else. Im not fussed about it myself but this system is about to go live and the customers first comment will be "whats that [1] doing next to the updates ?"  I'll have to look to see if I can see where its coming from to make its decision, as I say there are only 3 items - Statamic, a bard colour plugin and an unlisted eloquenty plugin (which has never caused an issue before and the version number installed is the same as the currently available version number) so at a bit of a loss but I'll have to check what its doing to make its [1] decision and take it from there. Thanks.

**Reply by jesseleite** [reply]
@tonybyng It's also worth noting the updates count is cached. If you `php artisan cache:clear`, does that resolve the issue?

**Comment by jasonvarga** [comment]
What does `php please support:details` output?

**Comment by tonybyng** [comment]
Hi

Sorry for the delay

/usr/local/php80/bin/php please support:details
Statamic 3.2.19 Pro
Laravel 8.68.1
PHP 8.0.11
nfragkos/eloquenty 1.1.0
xndbogdan/statamic-bard-text-color 2.2

**Reply by tonybyng** [reply]
@jasonvarga 

Ive just run a new composer update to bring it up to the  latest statamic so now its dropped back from 2 to 1 again but everything is meant to be updated now and @jesseleite unfortunately no, a cache:clear hasn't solved it either. Any thoughts ? Thanks

**Reply by jasonvarga** [reply]
You're on Statamic 3.2.19, but there's 3.2.20 available. So that'd be why you're seeing it.

**Comment by tonybyng** [comment]
@jasonvarga Thanks for the reply. Unfortunately not. As per my my last comment, I ran another composer update which brought it back from 2 to 1 - as it had gone to 2 when you did the 3.2.20 release (thats what told me it wasnt a caching issue too) so Im now back to 1 item to update with everything allegedly up to date and back to square one with regard to not knowing why it thinks there is something to update. ;-(

**Comment by tonybyng** [comment, marked as answer]
Just for clarification - screen shot showing eloquenty is the latest version and screenshot showing statamic is latest version

![image](https://user-images.githubusercontent.com/9347905/140767819-bb231694-0e7a-46aa-8507-213b42813c73.png)
![image](https://user-images.githubusercontent.com/9347905/140767916-184923ef-620c-41e7-bb6e-a4d3824994db.png)

**Reply by jesseleite** [reply]
Ah good find! I believe this is a bug, will take a look. You can track progress here: https://github.com/statamic/cms/issues/4681
